frontends/ol-search-ui/src/util.tsx

```diff
@@ -8,7 +8,7 @@ import {
 } from "./interfaces"
 import React, { useState, useEffect } from "react"
 import { capitalize, emptyOrNil } from "ol-util"
-import LocaleCode from "locale-code"
+import ISO6391 from "iso-639-1"
```

Comment:
We had been using [locale-code](https://www.npmjs.com/package/locale-code) for language names. Swap to [iso-639-1](https://www.npmjs.com/package/iso-639-1?activeTab=readme). These packages are by the same author & `language-code depends` on `iso-639-1`, so we already had it as a dependency. 

The difference is:
- `iso-639-1` expects a language code... 'en', 'fr', etc
- `locale-code` expects a locale-code, `en-US`, `en-GB`, `zh-CN`, etc

**The reason to swap is:**
- we had been appending a country code 'US' to **all** language codes. That was arbitrary and confusing, but was required to `locale-code`.